docs/example-scenario/infrastructure/multi-tier-app-disaster-recovery.md

```diff
@@ -0,0 +1,105 @@
+2. [**Azure load balancer**][loadbalancer-docs] distributes inbound traffic according to rules and health probes. A load balancer provides low latency and high throughput, and scales up to millions of flows for all TCP and UDP applications. An internet load balancer is used in this scenario to distribute incoming client traffic to the web tier. An internal load balancer is used in this scenario to distribute traffic from the business tier to the back-end SQL Server cluster.
+3. [**Availability sets**][availability-set-docs]  ensure that the VMs you deploy on Azure are distributed across multiple isolated hardware nodes in a cluster. Doing this ensures that if a hardware or software failure within Azure happens, only a subset of your VMs are impacted and that your overall solution remains available and operational.
+4. [**Availability Zones**][azureaz-docs] is a high-availability offering that protects your applications and data from datacenter failures. Availability Zones are unique physical locations within an Azure region. Each zone is made up of one or more datacenters equipped with independent power, cooling, and networking. 
+5. [**Disaster recovery using Azure Site Recovery (ASR)**][azure-site-recovery-docs] allows you to replicate virtual machines to another Azure region for business continuity and disaster recovery needs. You can conduct periodic DR drills to ensure you meet the compliance needs. The VM will be replicated with the specified settings to the selected region so that you can recover your applications in the event of outages in source region.
+
+
+### Alternatives
+
+* Windows can easily be replaced by a variety of other operating systems as nothing in the infrastructure depends on the operating system.
+
+* [SQL Server for Linux][sql-linux] can replace the back-end data store.
+The database can be replcaed by any standard database application available.
+
+## Other considerations
+
+### Scalability
+
+You can add or remove VMs in each tier depending on your scaling requirements. As the scenario is using Load balancers, you can add more VMs in the tier without impacting the application uptime.
+
+For other scalability topics, see the [scalability checklist][scalability] in the Azure Architecture Center.
+
+### Security
+
+All the virtual network traffic into the front-end application tier is protected by network security groups. Rules limit the flow of traffic so that only the front-end application tier VM instances can access the back-end database tier. No outbound Internet traffic is allowed from the businees tier or database tier. To reduce the attack footprint, no direct remote management ports are open. For more information, see [Azure network security groups][nsg-docs].
+
+For general guidance on designing secure scenarios, see the [Azure Security Documentation][security].
+
+
+## Related resources
```

Comment:
Is there anywhere else you'd suggest the customer go?  Do we have any quickstarts that they can deploy or additional resources which would help build this out?